app/templates/dataset.html

```diff
@@ -15,124 +15,89 @@ <h2>CONP Portal | {{ data.title }} </h2>
-                    </pre>
-
-                    <xmp>
+      
 # Here are commands to download content from this dataset:
-~$ datalad install --source http://github.com/CONP-PCNO/conp-dataset
-~$ datalad get --no-data conp-dataset/{{data.id}}
-~$ cd conp-dataset/{{data.id}}
-# For datasets that requires login to the remote server use: git annex enableremote <remote-name>
-~$ datalad get <filepath>
-                    </xmp>
-                  </fieldset>
-                </div>
-             </div>
-             <div class="row" style="padding:20px;">
-                 {% if data.id == '0ea345b4-62cf-11e9-b202-52545e9add8e' %}
-                {% include "/fragments/preventad.html" %}
-                {% endif %}
-             </div>
-         {% else %}
-             <p>No information available</p>
-         {% endif %}
-    </div>
- </div>
+    ~$ datalad install --source http://github.com/CONP-PCNO/conp-dataset
+    ~$ datalad get --no-data conp-dataset/{{data.id}}
+    ~$ cd conp-dataset/{{data.id}}
+
+# For datasets that requires login to the remote server use:
+    ~$ git annex enableremote &ltremote-name&gt
+    ~$ datalad get &ltfilepath&gt
+    </pre>
+  </div>
+</div>
 
 <script type="text/javascript">
   const reactElement = React.createElement(CONPReact.DatasetElement, {
     authorized: {{ "true" if data.authorized else "false" }},
-    id: '{{ data.id }}',
+  id: '{{ data.id }}',
     title: '{{ data.title }}',
-    isPrivate: {{ "true" if data.isPrivate else "false" }},
-    thumbnailURL: '{{ data.thumbnailURL }}',
+      isPrivate: {{ "true" if data.isPrivate else "false" }},
+  thumbnailURL: '{{ data.thumbnailURL }}',
     downloads: {{ data.downloads }},
-    views: '{{ data.views  }}',
+  views: '{{ data.views  }}',
     likes: '{{ data.likes }}',
-    dateAdded: '{{ data.dateAdded }}',
-    dateUpdated: '{{ data.dateUpdated }}',
-    size: '{{ data.size }}',
-    files: '{{ data.files }}',
-    subjects: '{{ data.subjects }}',
-    format: '{{ data.format }}',
-    modalities: '{{ data.modalities }}',
-    sources: '{{ data.sources }}',
-    imagePath: "/static/img",
-    downloadPath : '{{ data.downloadPath }}',
-    conpStatus : '{{ data.conpStatus }}'
+      dateAdded: '{{ data.dateAdded }}',
+        dateUpdated: '{{ data.dateUpdated }}',
+          size: '{{ data.size }}',
+            files: '{{ data.files }}',
+              subjects: '{{ data.subjects }}',
+                format: '{{ data.format }}',
+                  modalities: '{{ data.modalities }}',
+                    sources: '{{ data.sources }}',
+                      imagePath: "/static/img",
+                        downloadPath : '{{ data.downloadPath }}',
+                          conpStatus : '{{ data.conpStatus }}'
```

Comment:
Is the stair here required (desired)? 
Also, `downloadPath` and `conpStatus` have a space before column unlike others

Reply:
It's not required, the autoformatter likely did this. I'll tidy it up manually.